### ISSUE TITLE
Ditto is the best thing since sliced bread, just had to say it

### ISSUE THREAD

**Issue by shodanx2** [sabrogden/Ditto#741]
Please this issue open, it will remain relevant forever, why don't nearly all computer have clipboard managers that make sense.

Why would you want a windows 3.11 style clipboard that can only remember one thing at a time ! Disgraceful !

WIth computers you can remember even 7 things at a time, what a concept.

**Comment by satabol** [comment]
Ditto used on every computer I worked.

**Comment by shodanx2** [comment]
Did you know it can directly copy paste files (and all other things) accross the network to another computer ?

How cool is that, it even works across the internet

**Comment by Natas0007** [comment]
I fully agree it is absolutely awesome!  I've been using it for years!  However, I've recently made the move to a Windows ARM laptop, which has been great, but I'm unable to install/use Ditto.  I've been trying to get by with the built in Windows clipboard history, but it's UI is bad, doesn't allow search, and doesn't persist over reboots.  I'm sure you're aware, but below is from the log when trying to install via `winget`.

Is there any hope for an ARM version?

```
Compatibility mode: Yes (DetectorsAppHealth Installer)
Windows version: 10.0.26100  (NT platform: Yes)
64-bit Windows: Yes
Processor architecture: ARM64
User privileges: Administrative
Administrative install mode: Yes
Install mode root key: HKEY_LOCAL_MACHINE
64-bit install mode: No
Defaulting to OK for suppressed message box (OK):
    This program can only be installed on versions of Windows designed for the following processor architectures: x64
```

**Comment by sabrogden** [comment]
I thought windows should emulate most aps on arm.  Try using 32 bit https://github.com/sabrogden/Ditto/releases/download/3.24.246.0/DittoPortable_3_24_246_0.zip

**Comment by Chealer** [comment]
@Natas0007: That is tracked in ticket #950.

**Comment by Natas0007** [comment]
Thanks @Chealer! I have it working perfectly on arm now!
